content/docs/reporting/analytics-report.md

```diff
@@ -86,16 +86,22 @@ Save your report for quick access in the future and to avoid reconfiguring your
 
 #### Preset Reports
 
-Your account comes with several preset saved reports which are available from the report dropdown. All preset reports are also available through a menu by clicking the “View All Reports” button in the upper right-hand corner.
+Your account comes with several preset saved reports which are available from the sidebar. All preset reports are also available through a menu by clicking the All Reports” button at the bottom of the sidebar.
```

Comment:
consider adding that you can use a preset report as-is or use it as a starting point for a custom report which you can then save/edit. that might be obvious, I'm not sure 🤷